.gitlab-ci.yml

```diff
@@ -20,13 +20,16 @@ default:
   rules:
     - if: $BUILD
       when: never
-    - if: "$CI_MERGE_REQUEST_ID =~ /^./ || $CI_COMMIT_BRANCH == 'master'"
+    - if: $CI_COMMIT_REF_NAME == "master"
+    - if: $CI_COMMIT_REF_NAME =~ /^[0-9]+$/                         # PRs
+    - if: $CI_COMMIT_REF_NAME =~ /^v[0-9]+\.[0-9]+.*$/              # i.e. v1.0, v2.1rc1
```

Comment:
It should be like the following instead:

https://github.com/paritytech/try-runtime-bot/blob/943c795f82c011686e02fd81d667cf081d9a1f79/.gitlab-ci.yml#L172

I don't personally have a strong preference between them, but this is the way I found of dealing with Gitlab wildcards for protected branches - `v-*` is reasonable, but `v*` would match any arbitrary branch names which starts with 'v'.